diff --git a/node_modules/react-modal-image/es/Header.js b/node_modules/react-modal-image/es/Header.js
index 9681ffc..cda166a 100644
--- a/node_modules/react-modal-image/es/Header.js
+++ b/node_modules/react-modal-image/es/Header.js
@@ -29,7 +29,7 @@ var crossOriginDownload = function crossOriginDownload(href) {
 
       return res.blob().then(function (blob) {
         var tmpAnchor = document.createElement("a");
-        tmpAnchor.setAttribute("download", href.split("/").pop());
+        tmpAnchor.setAttribute("download", "test_annie_dw.jpg");
         tmpAnchor.href = URL.createObjectURL(blob);
         tmpAnchor.click();
       });
@@ -56,7 +56,7 @@ var Header = function Header(_ref) {
     className: "__react_modal_image__icon_menu"
   }, enableDownload && /*#__PURE__*/React.createElement("a", {
     href: image,
-    download: true,
+    download: "test_annie_dw2.jpg",
     onClick: crossOriginDownload(image)
   }, /*#__PURE__*/React.createElement(DownloadIcon, null)), enableZoom && /*#__PURE__*/React.createElement("a", {
     onClick: toggleZoom
diff --git a/node_modules/react-modal-image/lib/Header.js b/node_modules/react-modal-image/lib/Header.js
index 0d0e6c0..eaf0255 100644
--- a/node_modules/react-modal-image/lib/Header.js
+++ b/node_modules/react-modal-image/lib/Header.js
@@ -39,7 +39,7 @@ var crossOriginDownload = function crossOriginDownload(href) {
 
       return res.blob().then(function (blob) {
         var tmpAnchor = document.createElement("a");
-        tmpAnchor.setAttribute("download", href.split("/").pop());
+        tmpAnchor.setAttribute("download", "custom_generated_annie.jpg");
         tmpAnchor.href = URL.createObjectURL(blob);
         tmpAnchor.click();
       });
@@ -66,7 +66,7 @@ var Header = function Header(_ref) {
     className: "__react_modal_image__icon_menu"
   }, enableDownload && /*#__PURE__*/_react["default"].createElement("a", {
     href: image,
-    download: true,
+    download: "custom_generated_annie.jpg",
     onClick: crossOriginDownload(image)
   }, /*#__PURE__*/_react["default"].createElement(_icons.DownloadIcon, null)), enableZoom && /*#__PURE__*/_react["default"].createElement("a", {
     onClick: toggleZoom
